resources/assets/components/InboxItem/InboxTile.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class InboxTile extends React.Component {
+  displayImage(photo_url) {
```

Comment:
@katiecrane yea i think it could be the same, or take in a property and then parses if it is `default` or not.